apps/frontend/src/libs/enums/query-parameter-name.enum.ts

```diff
@@ -1,6 +1,7 @@
 const QueryParameterName = {
 	PAGE: "page",
 	PAGE_SIZE: "pageSize",
+	SEARCH: "q",
```

Comment:
I'd make search param more semantic
```suggestion
	SEARCH: "seach"
```